src/Instrumentation/Psr3/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry PSR-3 auto-instrumentation
+
```

Comment:
Could you add a quick overview here of what the instrumentation does? (ie, automatically inject trace id and span id into log message context of any psr3 logger)

Reply:
updated.